pkg/kvm/metrics/metrics.go

```diff
@@ -0,0 +1,35 @@
+package metrics
```

Comment:
We can register and gather all the custom metrics we want. Need to check what the build-ins are but this is just a simple example.

Currently for `kvm-operator` we have metrics in there three places:
* [operatorkit specific ones](https://github.com/giantswarm/operatorkit/blob/c4021a5284c6a938a4b874725e825dfe713bb536/pkg/controller/metric.go)
* [version_bundle_version_total](https://github.com/giantswarm/kvm-operator/blob/ad40372197e6e2b81571ad3d545d2a6a5e257692/service/metric/metric.go)
* [creation/deletion timestamp](https://github.com/giantswarm/operatorkit/blob/c4021a5284c6a938a4b874725e825dfe713bb536/pkg/controller/collector/timestamp.go#L87)